configuration/errors.js

```diff
@@ -3,10 +3,14 @@ module.exports = {
     EMAIL_NOT_FOUND: "User not found",
     EMAIL_INVALID_FORMAT: "Email in invalid format",
     EMAIL_TAKEN: "User with specified email already exists",
-    
+    PASSWORDS_DO_NOT_MATCH: "Passwords do not match",
     PASSWORD_NOT_PROVIDED: "Password not provided",
     INCORRECT_PASSWORD: "Incorrect password",
     FAILED_TO_SEND_EMAIL: "Failed to send email",
-    
-    USER_ROLE_NOT_FOUND: "User role not found"
+    USER_ROLE_NOT_FOUND: "User role not found",
+    USER_NOT_FOUND: "User not found",
```

Comment:
you added two of the same error messages, you could unite it into one and use it throughout the whole project